lib/px/utils/pxclient.lua

```diff
@@ -131,14 +133,28 @@ function M.load(config_file)
 
         pxdata['details'] = details;
 
-        -- Experimental Buffer Support --
         buffer.addEvent(pxdata)
         -- Perform the HTTP action
         if buflen >= maxbuflen then
             _M.submit(buffer.dumpEvents(), px_constants.ACTIVITIES_PATH);
         end
     end
 
+    function _M.send_enforcer_telmetry(details)
+        local  enforcer_telemetry= {}
+
+        details.os_name = jit.os
```

Comment:
what's the `jit` object? is it guaranteed to exist in our runtime?

Reply:
its part of openresty so it exists